draft-ietf-quic-ack-frequency.md

```diff
@@ -411,18 +411,30 @@ controllers.
 ## Congestion Control
 
 A sender needs to be responsive to notifications of congestion, such as
-a packet loss or an ECN CE marking. Also, window-based congestion controllers
-that strictly adhere to packet conservation, such as the one defined in
-{{QUIC-RECOVERY}}, rely on receipt of acknowledgments to send additional data into
-the network, and will suffer degraded performance if acknowledgments are delayed
-excessively.
-
-To enable a sender to respond to potential network congestion, a sender SHOULD
-cause a receiver to send an acknowledgement at least once per RTT if there are
-unacknowledged ack-eliciting packets in flight. A sender can accomplish this by
-sending an IMMEDIATE_ACK frame once per round-trip time (RTT), or it can set the
-Ack-Eliciting Threshold and Request Max Ack Delay values to be no more than a
-congestion window and an estimated RTT, respectively.
+a packet loss or an ECN CE marking.  To enable a sender to respond to potential
+network congestion, a sender SHOULD cause a receiver to send an acknowledgement
+at least once per RTT if there are unacknowledged ack-eliciting packets in flight.
+A sender can accomplish this by sending an IMMEDIATE_ACK frame once per
+round-trip time (RTT), or it can set the Ack-Eliciting Threshold and
+Request Max Ack Delay values to be no more than a congestion window and an
+estimated RTT, respectively.
```

Comment:
```suggestion
a packet loss or an ECN CE marking. Therefore, {{Section 13.2.2 of QUIC-TRANSPORT}} requires
the receiver to "send an ACK frame after receiving at least two ack-eliciting packets".
This document overwrites this requirement by providing the sender a way to signal the needed
ACK frequency to the receiver. To enable a sender to respond to potential
network congestion in a timely fashion, usually at least one acknowledgement
per RTT is needed if there are unacknowledged ack-eliciting packets in flight.
A sender can accomplish this by sending an IMMEDIATE_ACK frame once per
round-trip time (RTT). Further, a sender SHOULD  NOT set the Ack-Eliciting Threshold and
Request Max Ack Delay values to be more than a congestion window and an
estimated RTT, respectively.
```

Reply:
/the sender a way to signal/the sender with  a way to signal/

Reply:
Ian and I discussed today about the new "SHOULD NOT" in the proposed text. During discussion we reached the conclusion that it's better to not use normative language here. however, the more I think about it, I think it's actually important to make a recommendation about the boundary on how to set the values.

However, maybe this is just not the right place in the document? This paragraph discusses how a data sender can use this extension to ensure that there is at least one ack RTT. However, if we want to have normative language, maybe this should be added to the ACK_FREQUENCY Frame section?

Reply:
```suggestion
a packet loss or an ECN CE marking. Therefore, {{Section 13.2.2 of QUIC-TRANSPORT}} 
indicates "A receiver SHOULD send an ACK frame after receiving at least two ack-eliciting packets".
This document overwrites this requirement by providing the sender with a way to signal its preferred
ACK frequency to the receiver. To enable a sender to respond to potential
network congestion in a timely fashion, usually at least one acknowledgement
per round-trip time (RTT) is needed if there are unacknowledged ack-eliciting packets in flight.
A sender can accomplish this by sending an IMMEDIATE_ACK frame once per
RTT. Similarly, setting the Ack-Eliciting Threshold to a lower value than the current congestion window or
the Request Max Ack to less than the estimated RTT are two options to request at least one ACK frame per RTT.
However, the congestion window particularly but also the RTT are dynamic and therefore might require frequent updates if the selected value are close to this limit.
```
This removes normative language and adds even more explanation... Sorry for making the text longer but given this is a discussion section, maybe more words actually have to make it clearer.